README.md

```diff
@@ -13,53 +20,108 @@ The application stack consists of the following components:
+For this, you will need to have the following installed on your local machine:
 
-There are a number of Python libraries that you also need to run the application, including `flask`, `sqlalchemy`, and `sqlalchemy-cockroachdb`. Rather than downloading these dependencies directly from PyPi to your machine, you should list them in dependency configuration files (see [Local Deployment](#local-deployment) and [Multi-region deployment](#multi-region-deployment) for examples).
+- [Google Cloud SDK](https://cloud.google.com/sdk/install)
+- [Docker](https://docs.docker.com/v17.12/docker-for-mac/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
 ## Local Deployment
 
-### Database setup
+### Database and Environment Setup
 
-In production, you want to start a secure CockroachDB cluster, with nodes on machines located in different areas of the world. For debugging purposes, you can just use the [`cockroach demo`](https://www.cockroachlabs.com/docs/stable/cockroach-demo.html) command. This command starts up an insecure, virtual nine-node cluster.
+In production, you want to start a secure CockroachDB cluster, with nodes on machines located in different areas of the world. For debugging purposes, you can just use the [`cockroach demo`](https://www.cockroachlabs.com/docs/stable/cockroach-demo.html) command. 
+The steps below walk you through how to start up an insecure, virtual nine-node cluster.
 
-1. Run `cockroach demo`, with the `--nodes` and `--demo-locality` flags. The database schema provided in this repo assumes the [GCP region names](https://cloud.google.com/about/locations/). 
+> Note: Shutting down a cluster that was run in demo mode erases all data in the database. 
+> Because demo mode doesn't allow you to specify a specific database port, you will need 
+> to rerun steps 1-3 below upon restarting `cockroach demo` to reinitialize your database 
+> and environment.
+
+1. Run `cockroach demo` in `--insecure` mode with the `--nodes` and `--demo-locality` flags. The database schema provided in this repo assumes the [GCP region names](https://cloud.google.com/about/locations/). 
 
     ~~~ shell
-    $ cockroach demo \
+    $ cockroach demo --insecure \
     --nodes=9 \
     --demo-locality=region=gcp-us-east1:region=gcp-us-east1:region=gcp-us-east1:region=gcp-us-west1:region=gcp-us-west1:region=gcp-us-west1:region=gcp-europe-west1:region=gcp-europe-west1:region=gcp-europe-west1
     ~~~
 
+    Once the database finishes initializing, you should see a SQL shell prompt:
+    
     ~~~
     root@127.0.0.1:<some_port>/movr> 
     ~~~
 
     Keep this terminal window open. Closing it will shut down the virtual cluster.
+    
+    Also take note of the value for `<some_port>`, as you'll be using that for the next step.
+
+1. Configure environment variables
+     
+    To initialize the database and configure your `.env` file, we'll want to save
```

Comment:
It might be a little clearer to not mention the .env file yet, since we aren't touching it in this step. For example:

"Save the port from the previous step into a shell environment variable named `MOVR_PORT:
... "